### PR TITLE
CHE-2403 fix theia notification styles

### PR DESCRIPTION
**Pull request by olexii4**
Fix theia-Notification styles(Add limit for the maximum size).

Related issue: https://github.com/theia-ide/theia/issues/2403

selection_012

Signed-off-by: Oleksii Orel <oorel@redhat.com>

### CODE REVIEW

**Comment by akosyakov** [comment]
@AlexTugarev can you review?

File: packages/messages/src/browser/style/notifications.css [hunk 1]
```diff
@@ -44,7 +44,7 @@
     clear: both;
     box-sizing: border-box;
     position: relative;
-    width: 60%;
+    width: calc(100% - 10px);
```
Comment (AlexTugarev) [inline packages/messages/src/browser/style/notifications.css]: please revert this. it shouldn't have that width.

**Comment by AlexTugarev** [comment]
@olexii4 your changes do not work with regular messages. Also, it's not meant to let the notification grab all available space, but indeed scrolling is broken. 

I tried, and I think we should go with this for now. Would be nice, if you could incorporate and verify this.

```css
.theia-Notification {
    pointer-events: all;
    -webkit-user-select: none;
    -moz-user-select: none;
    -ms-user-select: none;
    user-select: none;
    display: flex;
    flex-direction: row;
    -webkit-justify-content: center;
    justify-content: center;
    clear: both;
    box-sizing: border-box;
    position: relative;
    width: 60%;
    min-width: 500px;
    background-color: var(--theia-ui-dialog-color);
    box-shadow: 0 5px 5px -5px rgba(0, 0, 0, 0.6);
    min-height: 35px;
    -webkit-animation: fadeInDown 0.5s ease both;
    animation: fadeInDown 0.5s ease both;
    margin-bottom: 1px;
    color: var(--theia-ui-dialog-font-color);
    max-height: 350px;
    /* overflow: hidden; */
}
```

```css
.theia-Notification .text {
    order: 2;
    display: block;
    /* max-height: calc(100vh - 100px); */
    /* max-width: calc(100vh - 50px); */
    -webkit-user-select: text;
    -moz-user-select: text;
    -ms-user-select: text;
    user-select: text;
    /* align-self: center; */
    flex: 1;
    /* height: 100%; */
    padding: 10px;
    overflow-y: scroll;
}
```

```css
.theia-Notification .text > p {
    margin: 0px;
    font-size: var(--theia-ui-font-size1);
    vertical-align: middle;
}
```

Regular messages: 

screen shot 2018-07-20 at 17 23 27

Log files:

screen shot 2018-07-20 at 17 22 36


I'm working currently on #1360, and I didn't consider scrolling there either. :-(

**Comment by AlexTugarev** [comment]
@olexii4, this is confusing. what kind of message are you referring to by the screenshot from above?

**Comment by olexii4** [comment]
@AlexTugarev 
Sorry. I added 'width: calc(100% - 10px); and overflow: scroll; '
just for test and forgot it there.
Only this:
    max-height: calc(100vh - 100px);
    max-width: calc(100vh - 50px);
    overflow: auto;

File: packages/messages/src/browser/style/notifications.css (hunk 1 above)
Comment (olexii4) [reply packages/messages/src/browser/style/notifications.css]: added accidentally

**Comment by olexii4** [comment]
@AlexTugarev I am testing your styles.

**Comment by olexii4** [comment]
Done

**Comment by olexii4** [comment]
Further improvements wouldn't be urgent

**Comment by olexii4** [comment]
@AlexTugarev Last variant. Please, have a look.
WDYT?

File: packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ import {injectable} from 'inversify';
 import {Message} from '@phosphor/messaging';
 import {Key} from '@theia/core/lib/browser';
 import {AbstractDialog} from '@theia/core/lib/browser/dialogs';
+import {NOTIFICATION, ICON, TEXT} from '@theia/messages/lib/browser/notifications';
```
Comment (AlexTugarev) [inline packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts]: I'm sorry, that should not be exported, nor reused. Please inline those styles for plugin-ext if you need something like this. They are subject to change in #1360.

BTW, this slipped in a review, as such constants should be encapsulated in a namespace. Also styles shouldn't be reused in such a way in general. You can e.g. have a different implementation for `MessageService` and replace the `messages` extension.

Comment (olexii4) [reply packages/plugin-ext/src/main/browser/dialogs/modal-notification.ts]: Totaly agree.  I was confused with export there(thought about local rules)
https://github.com/theia-ide/theia/blob/master/packages/messages/src/browser/notifications.ts#L18

**Comment by olexii4** [comment]
@AlexTugarev Done. WDYT?

**Pull request merged**